class.php

```diff
@@ -101,9 +96,12 @@ protected function _setPageLanguage(Page $page, string $languageCode)
     $kirby->site()->visit($page, $languageCode);
   }
 
-  protected function _generatePage(Page $page, string $path)
+  protected function _generatePage(Page $page, string $path, string $baseUrl)
   {
     $html = $page->render();
+    $html = str_replace("$this->_originalBaseUrl/", $baseUrl, $html);
```

Comment:
Maybe the following option is a bit easier to read:
```suggestion
    $html = str_replace($this->_originalBaseUrl . '/', $baseUrl, $html);
```